.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+sudo: required
+services:
+  - docker
```

Comment:
@amcmurray-bw am I remembering wrongly or didn't this work for you? Didn't you have to do this workaround? 🤔

Reply:
Ideally this is how you'd do it I believe, but it didn't grab the right version of docker when I did it for some reason.  [This](https://docs.travis-ci.com/user/docker/#installing-a-newer-docker-version) is what I did.